subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/internal/DefaultJvmTestSuite.java

```diff
@@ -137,7 +139,13 @@ public DefaultJvmTestSuite(String name, ConfigurationContainer configurations, D
         this.targets = getObjectFactory().polymorphicDomainObjectContainer(JvmTestSuiteTarget.class);
         this.targets.registerBinding(JvmTestSuiteTarget.class, DefaultJvmTestSuiteTarget.class);
 
-        this.dependencies = getObjectFactory().newInstance(DefaultJvmComponentDependencies.class, implementation, compileOnly, runtimeOnly, annotationProcessor);
+        this.dependencies = getObjectFactory().newInstance(
+            DefaultJvmComponentDependencies.class,
+            new DefaultDependencyAdder(factory, implementation),
```

Comment:
I think this is supposed to be like the others?

Reply:
Yes, fixed.